package.json

```diff
@@ -7,11 +7,26 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "prepare": "husky install"
```

Comment:
Github is wired, those changes already exist in master, why it's treating them as they are coming from your branch.

Reply:
It happened in Maher's branch too, I guess it's the CRLF shit